content/guides/tutorials/troubleshooting/ts-how-to-increase-max-upload-file-size/index.md

```diff
@@ -6,7 +6,7 @@ dnnversion: 09.02.00
 related-topics: ts-error-login-ip-filtering-is-currently-disabled,ts-error-another-user-has-taken-action-on-the-page,ts-error-unknown-server-tag-DNNComboBox,ts-error-could-not-load-awssdk,ts-error-sql-timeout,ts-error-argumentnullexception-after-move-upgrade,ts-install-missing-resources,ts-mixed-content-ssl,ts-broken-profile-image,ts-page-remains-in-draft,ts-unable-to-remove-page-redirect-urls,ts-site-theme-not-loading,ts-incomplete-content-localization,ts-missing-persona-bar
 ---
 
-# How to: Increase the Maximum File Size for Upload
+# How to: Increase the Maximum File Size for Upload in DNN CMS
```

Comment:
Can you use the official name "DNN Platform"?

Reply:
You are right, thanks for the feedback

Reply:
BTW I used DNN CMS because that seems to rank better, but you are still right :-)